docs/support/tutorials/python-usage-guide.md

```diff
@@ -0,0 +1,291 @@
+# Using Python on CSC supercomputers
+
+Some important aspects of working with the Python programming language
+are notably different on CSC supercomputers compared to usage on a personal
+device or in other HPC environments. To make the most of the computational
+resources available to you, it is helpful to be aware of the differences.
+
+See the
+[Python application page](../../apps/python.md)
+for general information on the Python language and
+pre-installed Python environments on CSC supercomputers.
+
+## Creating and managing Python environments
+
+### Installing Python packages to existing modules
+
+If there is a CSC-provided module that covers _almost_ everything you
+need, but it is missing a few Python packages, you can try installing
+those yourself with the pip package manager.
+
+See the [package lists on our Python application
+page](../../apps/python.md#pre-installed-python-environments) to find out
+which packages are installed in existing modules.
+If you think that some important package should be included by default
+in a module provided by CSC, don't hesitate to contact our [Service
+Desk](../contact.md).
+
+=== "Using `venv`"
+	!!! default ""
+		The recommended way to add packages on top of an existing environment
+		is to use [venv](https://docs.python.org/3/tutorial/venv.html), which
+		is a standard Python module for creating a lightweight "virtual
+		environment". You can have multiple virtual environments, for example
+		one for each project.
+
+		For example to install a package called `whatshap` on top of the
+		CSC-provided [python-data](../../apps/python-data.md) module:
+
+		```bash
+		cd /projappl/<your_project>  # change this to the appropriate path for your project
+		module load python-data
+		python3 -m venv --system-site-packages <venv_name>
+		source <venv_name>/bin/activate
+		pip install whatshap
+		```
+        Unlike for example Tykky, `venv` creates a new directory for the
```

Comment:
There needs to be an empty line after code blocks. Otherwise the following text will not be placed inside a `<p>` block, but as loose text.

```suggestion
		```

        Unlike for example Tykky, `venv` creates a new directory for the
```

Reply:
There is now an empty line after each code block